api_tests/users/views/test_user_settings.py

```diff
@@ -309,6 +310,13 @@ def test_create_email_invalid_format(self, app, url, payload, user_one):
         assert res.status_code == 400
         assert res.json['errors'][0]['detail'] == 'Enter a valid email address.'
 
+    def test_create_blacklisted_email(self, app, url, payload, user_one):
+        new_email = 'freddie@{}'.format(BLACKLISTED_DOMAINS[0])
```

Comment:
Very nice 👍 We have old tests (which I believe I wrote...) that could easily break since they rely on a specific domain being in the blacklist.